content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -360,15 +350,15 @@ sensuctl event info sensu-go-sandbox curl_timings --format json | jq .
 ...
 {{< /highlight >}}
 
-Because we configured a metric format, the Sensu agent is able to convert the Graphite-formatted metrics provided by the check command into a set of Sensu-formatted metrics (not shown in the output), which are then sent to the InfluxDB handler that reads Sensu-formatted metrics and converts them to a format InfluxDB accepts.
-Metric support isn't limited to just Graphite; the Sensu agent can extract metrics in multiple line protocol formats, including Nagios performance data.
+Because you configured a metric format, the Sensu agent can convert the Graphite-formatted metrics provided by the check command into a set of Sensu-formatted metrics (not shown in the output). These metrics are then sent to the InfluxDB handler, which reads Sensu-formatted metrics and converts them to a format InfluxDB accepts.
```

Comment:
```suggestion
Because the check definition specified a metric format of `graphite_plaintext`, the Sensu agent will treat the output of the check command as Graphite-formatted metrics and translate them into a set of Sensu-formatted metrics (not shown in the output). These metrics are then sent to the InfluxDB handler, which reads Sensu-formatted metrics and converts them to a format InfluxDB accepts.
```

Reply:
Yes, much better, thank you!